src/app/members/components/MemberDetails.tsx

```diff
@@ -11,117 +11,118 @@ import {
-              <h2
-                data-testid="user-name"
-                className="text-sm font-semibold text-gray-700"
-              >
-                {userInfo.name}
-              </h2>
-              <UserStatusBadge status={userInfo.status} />
-            </div>
+            <h2 data-testid="user-name" className="text-base text-gray-800">
+              {userInfo.name}
+            </h2>
+          </div>
+          <div className=" pl-14">
+            <UserStatusBadge status={userInfo.status} />
           </div>
-          {userInfo?.role === MemberRole.Incharge && (
-            <div className="mt-4 md:mt-0 flex items-center space-x-2 text-gray-700">
-              <ShieldIcon className="w-5 h-5" />
-              <span className="text-sm font-semibold">
-                CNIC: {userInfo.cnic}
-              </span>
+        </div>
+
+        {userInfo?.role === MemberRole.Incharge && (
+          <div className="flex items-center gap-3 text-gray-700">
+            <div className="flex items-center justify-center w-10 h-10 rounded-full bg-gray-100">
+              <ShieldIcon className="w-4 h-4 text-gray-700" />
             </div>
-          )}
-        </CardHeader>
-        <CardContent className="grid grid-cols-1 md:grid-cols-2 gap-8 px-6 pb-8">
-          <div className="space-y-6 text-gray-700">
-            <div className="space-y-3">
-              <h3 className="flex items-center space-x-2 text-sm font-semibold ">
-                <UserIcon className="w-5 h-5" />
-                <span>Personal Information</span>
-              </h3>
-              <div className="pl-7 space-y-2">
-                <p className="text-xs">
-                  <span className="font-medium text-muted-foreground">
-                    {" "}
-                    Age:{" "}
-                  </span>{" "}
-                  {userInfo.date_of_birth} years
-                </p>
-              </div>
-              <div className="pl-7 space-y-2">
-                <p className="text-xs">
-                  <span className="font-medium text-muted-foreground">
-                    {" "}
-                    Role:{" "}
-                  </span>{" "}
-                  {userInfo.role}
-                </p>
+            <span className="text-sm font-medium">CNIC: {userInfo.cnic}</span>
+          </div>
+        )}
+
+        <div className="text-gray-700">
+          <div className="space-y-1">
+            <h3 className="flex items-center gap-2 text-sm font-semibold">
+              <div className="flex items-center justify-center w-10 h-10 rounded-full bg-gray-100">
+                <UserIcon className="w-4 h-4" />
               </div>
+              <span>Personal Information</span>
+            </h3>
+            <div className="pl-12 space-y-2 ">
+              <p className="text-xs">
+                <span className="font-medium text-muted-foreground">Age:</span>{" "}
+                {getAge(userInfo.date_of_birth)} years
```

Comment:
can we use it outside `p` tag?

Reply:
I think it's fine if i move it outside the <p> tag it disturbed the UI.